wgpu/src/lib.rs

```diff
@@ -4991,11 +4991,20 @@ impl<'a> Drop for QueueWriteBufferView<'a> {
 impl Queue {
     /// Schedule a data write into `buffer` starting at `offset`.
     ///
-    /// This method is intended to have low performance costs.
-    /// As such, the write is not immediately submitted, and instead enqueued
-    /// internally to happen at the start of the next `submit()` call.
-    ///
     /// This method fails if `data` overruns the size of `buffer` starting at `offset`.
+    ///
+    /// This does *not* submit the transfer to the GPU immediately. Calls to
+    /// `write_buffer` begin execution only on the next call to
+    /// [`Queue::submit`]. To get a set of scheduled transfers started
+    /// immediately, it's fine to call `submit` with no command buffers at all:
+    ///
+    /// ```no_run
+    /// # let queue: wgpu::Queue = todo!();
+    /// queue.submit(std::iter::empty());
```

Comment:
This sample code could be made shorter:

```suggestion
    /// queue.submit([]);
```

I don't know if there are any relevant performance differences, or if this should be considered good style or not.

Reply:
+1! IME, there's no performance difference, and if there is, that sounds like a bug against `rustc`. 😅

I'd definitely prefer this form for folks new to Rust and/or graphics. This is simply easier to parse for anybody that's not familiar with `std::iter` (most Rustacean coworkers I've had have seldom used it).

Reply:
Okay, I've made this change.